test/redirect-request.js

```diff
@@ -255,16 +255,24 @@ for (const factory of [
   t.test('should follow a redirect chain up to the allowed number of times', async t => {
     const server = await startRedirectingServer(t)
 
-    const { statusCode, headers, body: bodyStream, context: { history } } = await request(t, server, undefined, `http://${server}/300`, {
-      maxRedirections: 2
-    })
+    try {
+      const { statusCode, headers, body: bodyStream, context: { history } } = await request(t, server, undefined, `http://${server}/300`, {
```

Comment:
Good work with the tests, but let's create a new test and keep this one as is. Feel free to copy/paste this test and just change the name; but let's revert this current tests to its original form;